Configuration to disable schema auto creation by the client

gmcoringa opened the pull request:
This pr address https://github.com/hortonworks/registry/issues/621

raju-saravanan commented:
@gmcoringa Thanks for raising this PR. I have a couple of questions, may I know how this change is going to address your use case? If "schema.auto.create" is disabled then the lastest schema version is being utilized to serialize the data, this would only work if the schemas are backward compatible or fully compatible.

gmcoringa commented:
@raju-saravanan we have some problems with the latest version when the client tries to get the schema to perform the serialization, because some avro schemas wasn't compatible with the latest avro version (`1.9.1`)

One of them was because the get actually calls `addSchemaVersion`.  This PR tries to address this problem and for us its a good solution because the creation of schemas are done through schema-registry ui with backward compatibility.

Meanwhile we fixed this problem using an older version of schema-registry and fixing all the incompatible avro schemas, but this wasn't enough, because the call to  `addSchemaVersion` perform a validation against all schema versions, due the default validation level `ALL`, which we fixed updating all schemas with the validation level `LATEST`. 

Actually this PR isn't needed anymore, so if you think that do not fit in schema registry design, feel free to close.
